projects/docs/docs/commands/generate.md

```diff
@@ -14,18 +14,30 @@ and Xcode projects and workspaces as a implementation detail to edit your projec
 
 To generate the project in the current directory, youn can simply run:
 
-```
+```bash
 tuist generate
 ```
 
-The command accepts the `--path` argument, which can be used to generate the project in a different directory.
+Moreover, if external dependencies exist in the [cache](/building-at-scale/caching/), Tuist replaces them with their pre-compiled version.
+
+In large Xcode projects that contain many targets and schemes, Xcode can be slow indexing the project.
+Moreover, the build system, which needs to resolve implicit dependencies, might take longer to do so because there are more Xcode objects to analyze.
+This is **not ideal for developers' productivity** and for that reason Tuist allows users to focus on a specific target or set of targets.
+
+```bash
+tuist generate MyApp
+```
 
-There might be situations when you are only interested in generating the project in a given directory, and not the projects it depends on. For that, you can pass the argument `--project-only`.
+The command generates and opens an Xcode workspace where the targets and schemes that are not directly related to `MyApp` are removed.
+Moreover, if the direct and transitive dependencies exist in the [cache](/building-at-scale/caching/), Tuist replaces them with their pre-compiled version.
+Thanks to that developers can safely clean their Xcode environment because they'll only be building the target they are focusing on.
```

Comment:
Random thought: focus on a target could also be expressed by generating a target. Since we are dropping the `focus` command, I worry using the verb `focus` could lose some of the readers.